mailu/README.md

```diff
@@ -43,6 +43,7 @@
 | `webdav.enabled`                  | enable webdav server                 | `false`                                   |
 | `ingress.externalIngress`         | Use externally provided nginx        | `true`                                    |
 | `ingress.tlsFlavor`               | [Choose from these](https://mailu.io/1.7/compose/setup.html#tls-certificates)  | `cert`                                   |
+| `ingress.annotations`               | Annotations for the ingress resource, if enabled. Useful e.g. for configuring the NGINX controller configuration.  | `{}`                                   |
```

Comment:
One last change: the default is now nginx.ingress.kubernetes.io/proxy-body-size: "0", not {}

Reply:
Technically not really because `nginx.ingress.kubernetes.io/proxy-body-size: "0"` is not specified in the template files but in the example values. It would be the case if it was written this way:
```yml
metadata:
  name: {{ $fullname }}-ingress
{{- if .Values.ingress.annotations }}
  annotations:
{{ toYaml .Values.ingress.annotations | indent 4 }}
{{ else }}
  annotations:
    nginx.ingress.kubernetes.io/proxy-body-size: "0"
{{- end }}
  labels:
    app: {{ $fullname }}
    component: admin
```

Reply:
No, it's not the "example values", it's the "default values"

Reply:
The `values.yml` is commonly used as an example but that doesn't mean that those values are the default ones.
A default value is when an `else` condition or better `default` is specified in the template files. Here is an example for `default`:
```yml
drink: {{ .Values.favorite.drink | default "tea" | quote }}
```
Taken from the documentation: https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function

Anyway it's just for avoiding confusion. If you prefer to have `nginx.ingress.kubernetes.io/proxy-body-size: "0"` anyway in the README then I'll do it.

Reply:
Actually, helm _does_ use values.yaml from the chart as defaults (it's the first values file that's always read, see https://helm.sh/docs/chart_template_guide/values_files/).
So yes, I'd really like to have it in the README (as it's already there for other default values).

Reply:
What if one specify a custom values file using `--values` without any ingress annotation in it, wouldn't be no ingress annotations generated in the actual Kubernetes manifest files? If that's correct that's what I was referring to default values.

I modified to `nginx.ingress.kubernetes.io/proxy-body-size: "0"`

Reply:
No, the file passed with `--values` is merged with the default one

Reply:
Exception would be if the file would define `{}` as annotations

Reply:
Ok my bad from my part and thanks for the explanations.